nixos/packages/givc-gen-certs.nix

```diff
@@ -0,0 +1,67 @@
+{ pkgs }:
+pkgs.writeShellScriptBin "givc-gen-certs" ''
+          set -xeuo pipefail
+
+          if [ $# -eq 1 ]; then
+            target="$1"
+          else
+            echo "Usage: $0 <basedir>" >&2
+            exit 1
+          fi
+
+          acl_prefix="otherName.1:1.2.3.4.5.6;UTF8"
+
+          # Function to create key/cert based on IP and/or DNS
+          gen_cert(){
+              name="$1"
+              path="$target"/"$name"
```

Comment:
`/` is not part of a variable name, so just `"$target/$name"` works equally.

Reply:
Anyway, I rewrote (almost) this into rust just for weekend rust practice

Reply:
Not yet, but anyway it copy from main ghaf PR, using only for generate test snakeoils certs